src/DSharp.Compiler/Compiler/ExpressionBuilder.cs

```diff
@@ -694,10 +694,24 @@ private Expression ProcessConditionalNode(ConditionalNode node)
 
             Debug.Assert(objectExpression.EvaluatedType is ISymbolTable table);
 
-            ISymbolTable typeSymbolTable = objectExpression.EvaluatedType;
-            string memberName = ((NameNode)node.RightChild).Name;
+            TypeSymbol evaluatedType = objectExpression.EvaluatedType;
```

Comment:
This code is less than optimal, ideally I need to rewrite this whole chain to resolve methods in a consisitant way, with checks for extension methods etc.